File: Crassus.Native.iOS/IndexPathDelegate.m
//
//  IndexPathDelegate.m
//  TheObiG LLC
//
//  Created by SolomonRain on 2/16/16.
//  Copyright © 2016 Crassus. All rights reserved.
//

@import UIKit;

long CrassusGetIndexPathSection(void * indexPath)
{
    return ((__bridge NSIndexPath*)indexPath).section;
}

long CrassusGetIndexPathRow(void * indexPath)
{
    return ((__bridge NSIndexPath*)indexPath).row;
}

File: Crassus.Native.iOS/IndexPathDelegate.c
//
//  IndexPathDelegate.c
//  TheObiG LLC
//
//  Created by SolomonRain on 2/16/16.
//  Copyright © 2016 Crassus. All rights reserved.
//

#include "IndexPathDelegate.h"

long GetIndexPathSection(void * indexPath, ObjectLongNumberFunction function)
{
    return (*function)(indexPath);
}

long GetIndexPathRow(void * indexPath, ObjectLongNumberFunction function)
{
    return (*function)(indexPath);
}